specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/examples/event-grid-schema/chat_participant_removed_from_thread_with_user.json

```diff
@@ -17,6 +17,10 @@
         "communicationUser": {
           "id": "8:acs:0a420b29-555c-4f6b-841e-de8059893bb9_00000008-77c9-8785-1655-373a0d00885f"
         }
+      },
+      "metadata": {
+        "key": "value",
+        "description": "A map of data associated with the message"
```

Comment:
The 'message' was a sample from the message events, here it should be either 'thread' or 'user'